File: people_database/js/person_spec.js
var Person = require('./person');

describe('Person', function() {

  it('has attributes', function() {
    var person = new Person({
            first_name: "Jose",
            last_name:  "Buendia",
            email:      "jose@gmail.com",
            state:      "CA"
            });

    expect(person.first_name).toBe("Jose");
  });
});

File: people_database/clojure/people_database/spec/people_database/person_spec.clj
(ns people-database.person-spec
  (:require [speclj.core :refer :all]
            [people-database.person :refer :all]))

(describe "construct-person"
  (it "should give me back their info"
    (let [person
      (construct-person "Jose" "Tellez" "mexicorules@gmail.com" "VA")]
      (should= "Jose" (person :first-name))
      (should= "Tellez" (person :last-name))
      (should= "mexicorules@gmail.com" (person :email))
      (should= "VA" (person :state)
      ))))


(run-specs)
